src/proto/NULS.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package TW.NULS.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message TransactionCoinFrom {
```

Comment:
maybe CoinFrom -> Input?

Reply:
CoinFrom is a class type defined in NULS 2.0. If it is confuse, iI will change it to Input.